test/parallel_api/iterator/iterators.pass.cpp

```diff
@@ -167,12 +162,14 @@ struct test_zip_iterator {
     void operator()(::std::vector<T1>& in1, ::std::vector<T2>& in2) {
         //runtime call to check default constructor and increase code coverage
         oneapi::dpl::zip_iterator<decltype(in1.begin()), decltype(in2.begin())> b;
+        //runtime call to check constructor with variadic arguments and increase code coverage
+        oneapi::dpl::zip_iterator<decltype(in1.begin()), decltype(in2.begin())> c(in1.begin(), in2.begin());
 
         //runtime call to check copy assignable operator and increase code coverage
         b = oneapi::dpl::make_zip_iterator(in1.begin(), in2.begin());
         auto e = oneapi::dpl::make_zip_iterator(in1.end(), in2.end());
 
-        EXPECT_TRUE( (b+1) != e, "size of input sequence insufficient for test" );
+        EXPECT_TRUE( (c+1) != e, "size of input sequence insufficient for test" );
```

Comment:
`b` stands on begin, that's why I think it is better to leave `b` here.

As for `c`, we should check that such constructor not only creates an iterator, but also creates it properly so that we can get expected results when working with it. Let's check it with `all_off` at least.

Reply:
Done